compiler/luci-interpreter/src/kernels/FullyConnected.cpp

```diff
@@ -77,6 +84,9 @@ void FullyConnected::execute() const
     case DataType::U8:
       evalQuantized();
```

Comment:
How about make `evalQuantized()` as template function for types with assymetric quantization like `int8` and `uint8`? =)

If so, we could reduce code duplication and use this method as:
```cpp
switch (input()->element_type())
  {
    case DataType::U8:
      evalQuantized<uint8_t>();
      break;
    case DataType::S8:
      evalQuantized<int8_t>();
      break;
    case DataType::FLOAT32:
      evalFloat();
      break;
```

Reply:
This technique could be used for other operations, like Conv, DepthwiseConv, etc. Let's consider this refactoring after enabling minimal set (from #7677) of operations.